### ISSUE TITLE
Any plan to implement dataloader of CE model, like Kaldi

### ISSUE THREAD

**Issue by Justbreak** [k2-fsa/icefall#116]
I tried to perform a VAD model based on two-class CE loss, but found that data preparation was complicated, and conventional data preparation would consume a lot of storage space. Are there any plans to provide dataloader for CE training?

**Comment by danpovey** [comment]
So where would you plan to get the alignments from for CE loss?
Or would be the same for the entire utterances?

**Comment by Justbreak** [comment]
My idea is to obtain alignment results based on Kaldi's GMM model, and then use pytorch as a neural network training tool
